src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -375,43 +355,11 @@ protected function escapeParams($arrValues)
 	/**
 	 * Debug a query
 	 *
-	 * @param \Database\Result $objResult An optional result object
+	 * @deprecated Deprecated in Contao 4.0, to be removed in Contao 5.0.
```

Comment:
Maybe better note: `@deprecated Deprecated in Contao 4.0, to be removed in Contao 5.0. - No op implementation until then.`
To tell developers that it is doing nothing anymore ans is only preserved to keep ABI compatibility.